README.rst

```diff
@@ -336,6 +336,11 @@ Choose from the list of available fixers:
                          PHP keywords MUST be in lower
                          case.
 
+* **mb_str_functions**
```

Comment:
The name must be the same of https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2082 where I didn't receive feedback for that.
I chose that name for the similarity with real functions name.

I kindly ask the opinion of another maintener before changing it in two separate PRs